src/main/java/Duke.java

```diff
@@ -1,10 +1,92 @@
+import java.util.Scanner;
+
 public class Duke {
+    public static String indentation = "    ";
+    public static String doubleIndentation = indentation + "  ";
+    public static String tripleIndentation = doubleIndentation + "  ";
+    public static String horizontalLine = indentation
+            + "<------------------------------------------------------------>\n";
+    public static Task[] Tasks = new Task[100];
+    public static int numberOfTasks = 0;
+
+    public static void printStatement(String statement){
+        System.out.println(horizontalLine + indentation + "Here is yuqiaoluolong's Duke: \n"
+                + statement + horizontalLine);
+    }
+
     public static void main(String[] args) {
         String logo = " ____        _        \n"
                 + "|  _ \\ _   _| | _____ \n"
                 + "| | | | | | | |/ / _ \\\n"
                 + "| |_| | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        String greet = indentation + " Hello! I'm Duke\n"
+                + indentation + " What can I do for you?\n";
+        String bye = indentation + " Bye. Hope to see you again soon!\n";
+        String wrongWarning = "Sorry, I don't know what's your command. Can you type again?\n";
+        String inputCommand;
+        boolean isBye = false;
+        int doneNum;
+        String outputStatement;
+        String description;
+        String date;
+
+        printStatement("Hello from\n" + logo);          // greet in the beginning
+        printStatement(greet);
+
+        Scanner in = new Scanner(System.in);
+        while (!isBye) {
+            inputCommand = in.nextLine();
+            switch (inputCommand.trim()) {
```

Comment:
i like how you used switch / case statements here 👍

Reply:
Thank you for appreciation. What do you mean by saying extract out individual cases? I did not get it